dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -290,6 +291,24 @@ class BGPPeerMgr(Manager):
         vrf, nbr = key.split('|', 1)
         if key not in self.peers:
             cmd = None
+
+            if "local_addr" not in data:
+                syslog.syslog(syslog.LOG_ERR, 'Peer {}. Error in missing required attribute "local_addr"'.format(key))
```

Comment:
I think it's a warning here

Reply:
Thanks for your suggestion. Have changed this code.